LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2017 Hewlett-Packard Enterprise Development Company, L.P.
+© 2017 EntIT Software LLC, a Micro Focus company, L.P.

```

Comment:
change to 2018 everywhere, because we will not release the change by the end of the year